content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -0,0 +1,116 @@
+
+There are several ways of doing this:
+
+* Edit `/etc/kubernetes/manifests/kube-controller-manager.yaml` to add the arguments to the command.
+  Keep in mind that this may be overwritten during the next upgrade.
+* If you're creating a new cluster, you can use a kubeadm [configuration file][config]:
+  ```yaml
+  apiVersion: kubeadm.k8s.io/v1beta1
+  kind: ClusterConfiguration
+  controllerManager:
+    extraArgs:
+      cluster-signing-cert-file: /etc/kubernetes/pki/ca.crt
+      cluster-signing-key-file: /etc/kubernetes/pki/ca.key
+  ```
+* You can also upload a config file using [`kubeadm config upload from-files`][config-upload]
+[cert-manager-issuer]: https://cert-manager.readthedocs.io/en/latest/tutorials/ca/creating-ca-issuer.html
+[kcm]: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/
+[config]: https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1
+[config-upload]: https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-config/#cmd-config-from-file
+
+## Approving requests
+
+Unless you've set up an external signer such as [cert-manager][cert-manager], you will need to manually approve certificates. This is done using the [`kubectl certificates`][certs] command. 
+The `kubeadm` command will output the name of the certificate you need to approve, then block and wait for approval to occur:
+
+```shell
+$ sudo kubeadm alpha certs renew apiserver --use-api &
+[1] 2890
+[certs] certificate request "kubeadm-cert-kube-apiserver-ld526" created
+$ kubectl certificate approve kubeadm-cert-kube-apiserver-ld526
+certificatesigningrequest.certificates.k8s.io/kubeadm-cert-kube-apiserver-ld526 approved
+[1]+  Done                    sudo kubeadm alpha certs renew apiserver --use-api
+```
+
+You can view a list of pending certificates with `kubectl get csr`.
+
+[manage-tls]: https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/
+[cert-manager]: https://github.com/jetstack/cert-manager
+[certs]: https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#certificate
+
+# Using Certificate Requests with Kubeadm
+
+To better integrate with external CAs, kubeadm can also produce certificate signing requests (CSRs). 
+A CSR represents a request for a CA to produce a signed certificate for a client. 
+In kubeadm terms, any certificate that would normally be signed by an on-disk CA can be produced as a CSR instead, but CAs themselves cannot.
+
+## Requesting CSRs
+
+### kubeadm init 
+You can request an individual CSR with `kubeadm init phase certs apiserver --use-csr`. 
+The `--use-csr` flag can only be applied to individual phases; once [all certificates are in place][certs] you can run `kubeadm init --external-ca`.
+
+You can pass in a directory to output the CSRs to with `--csr-dir`. 
+If it is not specified, the default certificate directory (`/etc/kubernetes/pki`) will be used.
```

Comment:
If it is not specified, the default certificate directory (`/etc/kubernetes/pki`) will be used. - > If it is not specified, the default certificate directory (`/etc/kubernetes/pki`) is used. 

(Keeping the sentece in the present tense.)